File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/test/GetSharePanelCommand.java
package com.github.kotvertolet.youtubejextractor.models.test;

import com.google.gson.annotations.SerializedName;

public class GetSharePanelCommand{

	@SerializedName("commandMetadata")
	private CommandMetadata commandMetadata;

	@SerializedName("clickTrackingParams")
	private String clickTrackingParams;

	@SerializedName("webPlayerShareEntityServiceEndpoint")
	private WebPlayerShareEntityServiceEndpoint webPlayerShareEntityServiceEndpoint;

	public void setCommandMetadata(CommandMetadata commandMetadata){
		this.commandMetadata = commandMetadata;
	}

	public CommandMetadata getCommandMetadata(){
		return commandMetadata;
	}

	public void setClickTrackingParams(String clickTrackingParams){
		this.clickTrackingParams = clickTrackingParams;
	}

	public String getClickTrackingParams(){
		return clickTrackingParams;
	}

	public void setWebPlayerShareEntityServiceEndpoint(WebPlayerShareEntityServiceEndpoint webPlayerShareEntityServiceEndpoint){
		this.webPlayerShareEntityServiceEndpoint = webPlayerShareEntityServiceEndpoint;
	}

	public WebPlayerShareEntityServiceEndpoint getWebPlayerShareEntityServiceEndpoint(){
		return webPlayerShareEntityServiceEndpoint;
	}
}

File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/newModels/GetSharePanelCommand.kt
package com.github.kotvertolet.youtubejextractor.models.newModels

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize
import java.io.Serializable

@Parcelize
class GetSharePanelCommand : Parcelable, Serializable {
    @SerializedName("commandMetadata")
    var commandMetadata: CommandMetadata? = null

    @SerializedName("clickTrackingParams")
    var clickTrackingParams: String? = null

    @SerializedName("webPlayerShareEntityServiceEndpoint")
    var webPlayerShareEntityServiceEndpoint: WebPlayerShareEntityServiceEndpoint? = null
    override fun toString(): String {
        return "GetSharePanelCommand{" +
                "commandMetadata = '" + commandMetadata + '\'' +
                ",clickTrackingParams = '" + clickTrackingParams + '\'' +
                ",webPlayerShareEntityServiceEndpoint = '" + webPlayerShareEntityServiceEndpoint + '\'' +
                "}"
    }
}